### DISCUSSION TITLE
RTSP stream not automatically renewed

### DISCUSSION THREAD

**Discussion by qeroll** [bluenviron/mediamtx#2412] [Questions]
Hi,

I am using mediamtx to listen to an RTSP UDP stream and publish it to WebRTC.
When I open the stream from my webcam (rtsp://127.0.0.1:8554), then run mediamtx and only then open localhost:8888/webcam in my browser everything works fine.

However, when I run mediamtx and only afterwards open the stream, it never connects successfully.
Same happens if I restart the stream after the mediamtx app has already been running properly.

Am I missing something? Perhaps some configuration?
Any help will be much appreciated!

Example log for the above is attached. I start the stream properly, then the app. Afterwards I restart the stream but the app doesn't reconnect

<img width="721" alt="image" src="https://github.com/bluenviron/mediamtx/assets/118657264/9cbdce3f-149f-4d04-8333-b67b648fe935">

**Comment by aler9** [comment]
Hello, from your descriptions and from the partial logs it seems like the connection between the webcam and the server gets truncated after some seconds. If you still have problems, open an issue and follow instructions in order to provide data that allows to replicate the bug.

[a comment by an automated account is not shown]
